presto-docs/src/main/sphinx/release/release-0.263.rst

```diff
@@ -0,0 +1,23 @@
+=============
+Release 0.263
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a correctness bug in distinct aggregations when combined with a filtering mask containing null values.
+* Improve spilling CPU and memory efficiency for ``DISTINCT`` and ``ORDER BY`` aggregations.
+* Add a new session property ``heap_dump_on_exceeded_memory_limit_enabled`` to enable heapdump on exceeded memory failures. The heapdump file directory can be provided using ``exceeded_memory_limit_heap_dump_file_directory`` session property.
```

Comment:
The session property is not the feature rather the functionality is. Lets rephrase this as - 

```
Add support to create a heapdump when we exceed local memory limits for a query. This can be enabled with the ``heap....`` session property and the directory to create the heapdump can be set by ``....`` session property.
```